docs/modules/ROOT/pages/how-to/day2ops/configure-kibana.adoc

```diff
@@ -0,0 +1,28 @@
+= Configure Kibana Index Patterns
+
+This guide describes how to configure Kibana index patterns.
+This must be done once after installing the OpenShift EFK logging stack.
+
+== Prerequisites
+
+* Administrator access to the {zone}
```

Comment:
No, administrative access is not required. This can be done by all authenticated users.

Reply:
I'd have to test again. I could not do it with limited permissions. I'll try again to make sure.